files/acme-controller-template.yml

```diff
@@ -15,6 +15,12 @@ parameters:
   name: EXPOSER_IMAGE
 - description: Docker Image Tag of ACME challenge exposer
   name: EXPOSER_IMAGE_TAG
+- description: HTTP proxy host
```

Comment:
Is it really fair to take the assumption that the HTTP proxy is always equal to the HTTPS proxy?